packages/react-components/src/components/CameraButton.tsx

```diff
@@ -48,6 +48,16 @@ export interface CameraButtonStrings {
    * Tooltip of camera menu
    */
   cameraMenuTooltip: string;
+  /* @conditional-compile-remove(call-with-chat-composite) @conditional-compile-remove(control-bar-split-buttons) */
```

Comment:
These should only reference `control-bar-split-buttons` feature as the split buttons in `react-components` can be stabilized independently.

And any case, I think both of these features are now stabilized, so don't add conditional compilation at all: https://github.com/Azure/communication-ui-library/blob/main/common/config/babel/.babelrc.js#L30

Reply:
okay will remove these comments! 😊